include/oneapi/dpl/internal/random_impl/uniform_real_distribution.h

```diff
@@ -210,14 +210,16 @@ class uniform_real_distribution
     generate(_Engine& __engine, const param_type& __params)
     {
         auto __engine_output = __engine();
-        result_type __res;
+        result_type __res{};
```

Comment:
why do we need {} here?

Reply:
To not keep uninitialized values in the code